<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Participants</class>
 <widget class="QDialog" name="Participants">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load participans</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>861</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Dataset Structure</string>
   </property>
   <widget class="QComboBox" name="comboBoxType">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>30</y>
      <width>79</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Dataset Data type"/>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Data type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>34</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Session:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSession">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>off</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>34</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Task:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTask">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>off</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSuffix">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>none</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>33</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Suffix:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxExtension">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>30</y>
      <width>79</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="File extension"/>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="File extension"/>
    </property>
    <property name="text">
     <string>Extension:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDT">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Data type:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonShow">
    <property name="geometry">
     <rect>
      <x>714</x>
      <y>80</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Show participants</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDD">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Data description:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDatatype">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>---------</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDataDescrip">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>---------------------------------</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxParticipants">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>861</width>
     <height>481</height>
    </rect>
   </property>
   <property name="title">
    <string>Participants List</string>
   </property>
   <widget class="QTableWidget" name="tableParticipants">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>841</width>
      <height>421</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAll">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>25</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Check All</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonImport">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>653</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>653</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset theme="Cancel"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
